ansible/roles/etcd/defaults/main.yaml

```diff
@@ -45,4 +45,5 @@ etcd_listen_client_legacy_urls: "{{ etcd_url_scheme }}://0.0.0.0:{{ etcd_client_
 etcd_data_dir: /var/lib/etcd
 
 etcd_download_url_base: "https://github.com/coreos/etcd/releases/download/v{{ etcd_version }}"
-etcd_download_url: "{{ etcd_download_url_base }}/etcd-v{{ etcd_version }}-linux-amd64.tar.gz"
+etcd_download_file_name: "etcd-v{{ etcd_version }}-linux-amd64"
```

Comment:
Just for sake of consistency: ``s/file_name/filename``

Running ``grep -rn file_name`` over ansible dir, no match found. Running ``grep -rn filename`` a lot of occurrences found.